src/Nox.Types/Types/CurrencyCode3/CurrencyCode3.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Class for three-letters currency code (ISO 4217).
+/// </summary>
+public sealed class CurrencyCode3 : ValueObject<string, CurrencyCode3>
+{
+    private readonly string[] CurrencyCodes = Enum.GetNames(typeof(CurrencyCode));
```

Comment:
Can we put this as a private readonly static variable? Also, can we rename it using the following convention https://regusit.atlassian.net/wiki/spaces/NOX/pages/3153461266/Coding+Conventions+Styles+and+Guidelines#Private-static-fields ?